ush/forecast_predet.sh

```diff
@@ -208,8 +208,8 @@ FV3_GFS_predet(){
   print_freq=${print_freq:-6}
 
   #-------------------------------------------------------
-  if [ $CDUMP = "gfs" ] && [ $rst_invt1 -gt 0 ] && [ $MEMBER -lt 0 ]; then
-    RSTDIR_ATM=${RSTDIR:-$ROTDIR}/${CDUMP}.${PDY}/${cyc}/atmos/RERUN_RESTART
+  if [[ $CDUMP = "gfs" || $RUN = "gefs" ]] && [ $rst_invt1 -gt 0 ]; then
+    RSTDIR_ATM=${RSTDIR_ATM:-${RSTDIR:-$ROTDIR}/${CDUMP}.${PDY}/${cyc}/atmos/RERUN_RESTART}
```

Comment:
Do we need to have both `$RSTDIR` or `$RSTDIR_ATM`?

Reply:
I didn't find "RSTDIR=" in the g-w repo. However, I am not sure whether some others need it or not. So I keep it. If you think RSTDIR can be removed, then I will remove it.

Reply:
I don't see it either. Just remove it and only use RSTDIR_ATM

Reply:
Done